src/constants/links.js

```diff
@@ -1,13 +1,33 @@
 export const header = {
-  navDropDownLinks: [
-    ['link-one', 'link-two'],
-    ['link-three', 'link-four'],
+  navDropDowns: [
+    [
+      {
+        label: 'navigation.dropdowns.0.linkLabels.0',
```

Comment:
Now will the user be expected to update the labels in this file themselves when they add or subtract nav dropdowns?

Reply:
Exactly, which is why I thought we were hard-coding.

Reply:
Yeah... trying to think of other options

Reply:
We can discuss this with Rapi during the next code review.